src/metrics/test/metrics_test.cpp

```diff
@@ -369,5 +369,27 @@ TEST_CASE("Genny.ActiveActors metric") {
     }
 }
 
+TEST_CASE("Operation with threshold") {
+    RegistryClockSourceStub::reset();
+    auto metrics = v1::RegistryT<RegistryClockSourceStub>{};
+
+    genny::metrics::v1::ReporterT{metrics};
+
+    // Create an actor where 50% of the operations cannot exceed 10ns.
```

Comment:
maybe also a test where threshold is 0 or 100?

Reply:
Could also add a test with the first operations being slow ones. This would throw or not throw depending on what you think about my other comment about checking `> failedPercentageThreshold`.